pkg/envtest/server.go

```diff
@@ -147,8 +147,23 @@ type Environment struct {
 	// values are merged.
 	CRDDirectoryPaths []string
 
+	// DownloadBinaryAssets indicates that the envtest binaries should be downloaded.
+	// If this field is set:
+	// * DownloadBinaryAssetsVersion must also be set
+	// * If BinaryAssetsDirectory is also set, it is used to store the downloaded binaries,
+	//   otherwise a tmp directory is created.
+	DownloadBinaryAssets bool
+
+	// DownloadBinaryAssetsVersion is the version of envtest binaries to download.
+	DownloadBinaryAssetsVersion string
```

Comment:
We could maybe default this to the latest version?

Reply:
Good idea, done.

Reply:
I might be blind but I don't see it?

Reply:
Sorry, pushed just now :)